test/fixtures/jest/prefer-expect-assertions.fail.ts

```diff
@@ -1,3 +1,3 @@
 it('my second test', (): void => {
-  expect(1).toStrictEqual('foo');
+  expect(1).toBe('foo');
```

Comment:
`prefer-to-be` is a new rule in the recommended set from v25.0.0: https://github.com/jest-community/eslint-plugin-jest/blob/main/CHANGELOG.md#2500-2021-10-10